src/Themes/NexusMods.Themes.NexusFluentDark/Styles/Controls/Button/BareIconStyles.axaml

```diff
@@ -43,11 +43,6 @@
             <Setter Property="Size" Value="24" />
         </Style>
 
-        <Style Selector="^:pressed">
-            <!-- Accentuate the pressed animation -->
```

Comment:
This is used in the downloads (In Progress) page; and should not be removed.

Reply:
Requested by @captainsandypants, Updated with color states so we don't have to use transform scale

https://github.com/Nexus-Mods/NexusMods.App/assets/26166085/3cc7ea3f-7b57-4273-a085-c055f3a9f6ab